### PR TITLE
Bug 1252286 - Use Android's Wikipedia search icon for all locales

### PR DESCRIPTION
**Pull request by thebnich**
Removes the `wikipedia.xml` overlay and updates engines accordingly.

### CODE REVIEW

File: Client/Assets/Search/SearchPlugins/ar/wikipedia-ar.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
...amPD5z1WnNMb/dpW/uAdbipu413+qcl2H3N3cXFv9DNNtvmg68uxMY6dPfewTZNvOauouqEdULlTY532IgqHua+ZxnoHm4ANj63biRNQPZedVPY5nTR1BXsVbNuyTlsDqy4tW/wK4NYiJ01AXMlm+wGrL1cDnWbLejC4m7T4DvAOOSkD4v0qztGyLzpUweOk4UXAp6XV72DNmU7qgPPao+DJ0uLTFPzXXGN5I9acU1o8vb14jZzUAfFeBdvMtuwLDlfwIKme2248T1p9MvY59zACLqcy4Kf5eQr+ZbYRfrAFaXHaf/FRGUZAfE/Bzwf8OKwpuJ/0vYRNPEpavT248rACbqfimtLiqxT8w4yIz+Of5qTV7+LHMqyAaf9R8Clp8ex2K7iXuAa4q7R6aXCH4QXE4xVsOq+0+DYFfzXth/ilnrT6Tvzb9DADlqwqDPjTd377FHwI3EBaPb0NPFaGGRBvoLBiUVr8EFS+KgN8KpYtDjvgQo6i8Hhp8ZJUjnIZaXXGNrxKhh0QX6TwHzPS4uco/FtPWr0T9jnPKAJuOOiCc/UqVG4jrf4AH5dRBPT8UWHg3/BvUfhl635ZcNXRBMSDqFxFWvwklVu07O/BT2RUAad0GIWPtexXgMpPt+xntAt3Gl1AvJjCPufasn4d/6NyY2n4DPzXzCgDzmEvhVdIw+tj02X8nsIPGvus7Xi8jDIgPkLhSPON5ad4t7gbletJ9S71n+BIA65C5dGal1g3nENM+weF70j1R3iNjDogfkrhn6br5dU/4UXS94EtF7+vUC8IjjzgrlRuJ33v27j2dTLbKXxd+r4Pn5RRB9RfxsL3y4H/i0dI9TFUrirOZDeuPh4B8TQqVyjH/ZtZqZ7KIRS+JJ6Fn8k4BNRL9IUPmncwbr/1b37lanbiLuMTEG+nsNfb8cOWq0DLFA7E/8yMU8AlNLnaoP/alSfKOAXEN6h8Qlo8k3UKq5bGLeAWFPa6oLT6Ogqvk3ELmPJ3wOtlgOe0B2w6//gFxCPBsjPIQN8JPi3jGDDvSDxVDPaCNnFNOdGczrNzIrEnRyR5XPZlMIdmKofnpTnx6B783QV0ARNOF9AFdAFdQBfQBXQBXUAX0AV0AV3A6OgCVjLJrEzlj5lk/jiVt2eSefvEvzTJxL84zMS/PM//Af1pBRpQAkXBAAAAAElFTkSuQmCC</Image>
```
Comment (rnewman) [inline Client/Assets/Search/SearchPlugins/ar/wikipedia-ar.xml]: Does the `en` one overlay onto these? I'm concerned that `en` will get a 96x96 icon but every other locale will now fall back to 16x16!

The data URI looks bigger than 16x16, but not as big as 90x90…

Comment (thebnich) [inline Client/Assets/Search/SearchPlugins/ar/wikipedia-ar.xml]: It's confusing, but this smaller data URI, which is also incorrectly labeled as 16x16 in the `<Image>`, is actually a larger 96x96 icon (verifiable by copy/pasting the URI into the browser and viewing the image info).

Perhaps the Android PNGs are compressed?

Comment (rnewman) [inline Client/Assets/Search/SearchPlugins/ar/wikipedia-ar.xml]: Shouldn't these be labeled as 96*96?

Comment (thebnich) [inline Client/Assets/Search/SearchPlugins/ar/wikipedia-ar.xml]: Yep....I can file an l10n bug to have this fixed, though it doesn't make any practical difference for Android or iOS (other than confusing markup).

**Pull request merged**